src/components/data-exploration/embedding/Embedding.jsx

```diff
@@ -35,18 +34,22 @@ const Scatterplot = dynamic(
   { ssr: false },
 );
 
-const initialZoom = 4.00;
+const INITIAL_ZOOM = 4.00;
 const cellRadiusFromZoom = (zoom) => zoom ** 3 / 50;
 
+const EM = 16; // px
+const CELLINFO_WIDTH = 300; // px
+const CELLINFO_Y_PADDING = 6 * EM;
```

Comment:
some of these, like `EM` and `CELLINFO_Y_PADDING` are duplicates to the constants we have in CellInfo. Look at my other comment in this `Embedding.jsx` file, but these constants shouldn't be necessary in here.

Reply:
Moved into `CellInfo.jsx` as recommended

Reply:
yes, that's better